plot.py

```diff
@@ -3,12 +3,26 @@
 import matplotlib.pyplot as plt
 import argparse 
 from scipy import stats
+from quantilizer import ClassificationModel
+from wrappers import RobustRewardEnv
+from dataset import Dataset
+from sklearn.metrics import confusion_matrix
+from sklearn.model_selection import train_test_split
+import itertools
+from sklearn.decomposition import PCA
+import seaborn as sn
+import pandas as pd
+from matplotlib import cm
+import os
+from mpl_toolkits.mplot3d import Axes3D
+from collections import Counter
+from quantilizer import test
+from joblib import dump, load
+import os.path
+import time
 
 def load_ep_rets(filename, debug=False):
```

Comment:
is the `debug` argument still being used?

Reply:
Nice catch.